charts/wordpress/v2.1.10/templates/deployment.yaml

```diff
@@ -13,7 +13,6 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
-        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
Can we alter the existing chart? Or do we have to create a new version/folder. Assuming some people can already use it.